### ISSUE TITLE
Be careful using template literals on the front end

### ISSUE THREAD

**Issue by jwld** [facn2/PJS-FAC#16]
https://github.com/facn2/PJS-FAC/blob/master/scripts/index.js#L58

You can't rely on every user's browser being up to date enough to run ES6 JavaScript, so it's best not to use them on the front end. 

You can use them on the backend as much as you want however (next week...)

**Comment by ghost** [comment]
Agreed, however at this point we wouldn't know how to deal with that situation outside of using ES6.
